app/gui/StreamSegue.qml

```diff
@@ -137,6 +137,9 @@ Item {
 
             // Run the streaming session to completion
             session.exec(Screen.virtualX, Screen.virtualY)
+
+            // Show the Qt window again after streaming
+            window.showWindow()
```

Comment:
in my test, i3ipc was never trigged when disconnecting time.
it would be related QT rendering & event part.
but I give up to find real reason, just forceful wake up the application.
(it could be breaking change)